templates/v2/pro/rosalie/_rosalie_card.html.twig

```diff
@@ -0,0 +1,13 @@
+{% extends 'v2/common/_card.html.twig' %}
+
+{% block customContent %}
+    <div class="d-flex flex-column justify-content-center align-items-center font-size-1-2">
+        <p class="text-left col-11 py-2">{{ 'rosalie_link_text'|trans }}</p>
+        <div class="d-flex col-7 justify-content-start">
+            {{ form_start(form) }}
```

Comment:
Ton html est bizare, ton form end et for start devraient etre au meme niveau de hieérarchie dans le dom